ui/components/NotificationCenter/constants.js

```diff
@@ -37,4 +39,86 @@ export const SEVERITY_STYLE = {
     color : NOTIFICATIONCOLORS.WARNING
   },
 
+}
+
+//TODO: This should be generated from OPENAPI schema
+const EVENT_SCHEMA = {
+  type : "object",
```

Comment:
The schema is already present but the contract b/w fronted and backend is missing.

also, this schema is inaccurate as it lacks category and action